### PR TITLE
Add a new set of visualization APIs

### PR DESCRIPTION
**Pull request by hellock**

### CODE REVIEW

File: mmcv/visualization/figure.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            filepath (str): The filepath to be saved.
+        """
+        mmcv.imwrite(self._data, filepath)
+
+
+def plot_bboxes(bboxes, color='green', top_k=-1, thickness=1):
+    """Plot bboxes on a figure.
+
+    Args:
+        bboxes (ndarray): An ndarray of shape (n, 4) or (n, 5). The first 4
```
Comment (yhcao6) [inline mmcv/visualization/figure.py:107]: A ndarray

File: mmcv/visualization/figure.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+def plot_bbox_labels(bboxes,
+                     labels,
+                     class_names=None,
+                     color='green',
+                     score_thr=0,
+                     font_scale=0.5):
+    """Plot bbox labels on a figure.
+
+    Args:
+        bboxes (ndarray): An ndarray of shape (n, 4) or (n, 5). The first 4
```
Comment (yhcao6) [inline mmcv/visualization/figure.py:146]: A ndarray

File: mmcv/visualization/figure.py [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+            columns indicates (left, top, right, bottom), and the 5th column
```
Comment (yhcao6) [inline mmcv/visualization/figure.py:147]: indicate

File: mmcv/visualization/figure.py (hunk 1 above)
Comment (hellock) [reply mmcv/visualization/figure.py:107]: An ndarray

**Pull request closed without merging**